metau-capstone/Delegates/AppDelegate.m

```diff
@@ -33,6 +33,10 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     return YES;
 }
 
+- (BOOL)application:(UIApplication *)app openURL:(NSURL *)url options:(NSDictionary<UIApplicationOpenURLOptionsKey,id> *)options {
+    NSLog(@"open url through custom url scheme");
```

Comment:
Next week as we continue to do polish we can remove some of these logs and clean up our code a bit.